### PR TITLE
Add table export option

### LINKED ISSUE #128: Auto fill sequence change form
I'd like to add a new export option which would automatically fill out the table from this [form](https://icope.concordia.ca/content/documents/fileItemController/Sequence%20Change%20Form%20-%20TemplateAug2015.docx).

I'm thinking the best way to do it is to generate a filled clone of the table from the document using html and convert it to pdf with a tool like [wkhtmltopdf](https://wkhtmltopdf.org/). If that can be done, then we can change the export menu to provide the following options:

Export to:
- PDF (table)
- HTML (table)
- PDF (list)
- Plain Text (list)

Where table would be the new kind of export and list is the kind we're currently providing

### PR DESCRIPTION
**Pull request by Davidster**
resolves #128 

## Summary

As per #128, I used `wkhtmltopdf` to generate pdf files from html. I used the following [gist](https://gist.github.com/paulsturgess/cfe1a59c7c03f1504c879d45787699f5) to help me install the program onto our VM.

I added a new function to the export servlet which converts the sequence json into an html page with a table representing the sequence. It performs this conversion in a very similar way to how the markdown list is generated. 

I changed the server API a little to accommodate this change:

- Added an `exportType` parameter, which can use the values `table` or `list` so as to select which type of export is to be performed. 
- Added a `programName` paramater, whose value is essentially just added in bold to the top of the page regardless of whether you're exporting to a list or table. The wiki will be changed accordingly once this PR is merged.

~I will need to update our REST API wiki page accordingly once this is PR merged.~ wiki has been updated

### CODE REVIEW

File: src/main/webapp/exports/manway.txt [hunk 1]
```diff
@@ -8,4 +8,4 @@ manmanwei
 .
 .
 .
-For real though we need at least 1 file inside the exports folder otherwise the directory doesn't get created on the VM
\ No newline at end of file
+For real though we need at least 1 file inside the exports folder otherwise it doesn't get created on the VM
```
Comment (PeterGhimself) [inline src/main/webapp/exports/manway.txt]: check

Comment (Davidster) [reply src/main/webapp/exports/manway.txt]: ?

Comment (Davidster) [reply src/main/webapp/exports/manway.txt]: ^

Comment (Davidster) [reply src/main/webapp/exports/manway.txt]: ^

Comment (Davidster) [reply src/main/webapp/exports/manway.txt]: ^

Comment (Davidster) [reply src/main/webapp/exports/manway.txt]: ^

**Pull request merged**